cf_xarray/tests/test_helpers.py

```diff
@@ -0,0 +1,36 @@
+import numpy as np
+from numpy.testing import assert_array_equal
+
+import cf_xarray as cf  # noqa
+
+from .datasets import airds, mollwds
+
+
+def test_bounds_to_corners():
+    # 1D case
+    ds = airds.cf.add_bounds(["lon", "lat"])
+    lat_c = cf.bounds_to_corners(ds.lat_bounds, bounds_dim="bounds")
+    assert np.all(ds.lat.values + 1.25 == lat_c.values[:-1])
```

Comment:
Shouldn't this be `lat_c.values[1:]`? Can we also use `assert_array_equal`

Reply:
Latitudes are stored in decreasing order on that dataset, which explains the `[:-1]`.